src/utils/constants.ts

```diff
@@ -4,10 +4,9 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-
+import { CORE_EXTENSION_ID, SFDX_PROJECT_FILE } from 'mobile-lsp-server';
```

Comment:
I think you can do 
`export  { CORE_EXTENSION_ID, SFDX_PROJECT_FILE } from 'mobile-lsp-server';`
Can you give it a try?

Reply:
Good point! Change to below way since `SETTING_SECTION_DIAGNOSTIC ` still needs import
import { CORE_EXTENSION_ID } from 'mobile-lsp-server';
export const SETTING_SECTION_DIAGNOSTIC = `${CORE_EXTENSION_ID}.diagnostics`;